### PR TITLE
Add testkits infra

### PR DESCRIPTION
**Pull request by giladsegal**

### CODE REVIEW

File: packages/wsr-types/src/drivers/baseDrivers.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+declare namespace __WSRTests {
+  // TODO: types cannot be augmented, and I can't use
```
[the remark this answers is not in the document]
Comment (mosheKerbel) [reply packages/wsr-types/src/drivers/baseDrivers.d.ts]: @giladsegal suggestion - we can still use your types soultion, we can move `baseDrivers.d.ts` up one dir into a new one `baseDrivers`
then it won't be added multiple times by the concat commands.
we will then copy this `baseDrivers.d.ts` as it is
WDYT?

[the remark this answers is not in the document]
Comment (giladsegal) [reply packages/wsr-types/src/drivers/baseDrivers.d.ts]: The `concat` is not the problem.
When the client includes a single testing type definition files, it has to include the base driver types.
We can do it in two ways:
1. Create a separate type definitions file for the base drivers which will be included by the client
2. Include it in any type definitions file but then we can't use `type`

[the remark this answers is not in the document]
Comment (mosheKerbel) [reply packages/wsr-types/src/drivers/baseDrivers.d.ts]: Yes that's what I meant - option number 1. Not concat it to each teskit, but a new d.ts that the client will include.

[the remark this answers is not in the document]
Comment (giladsegal) [reply packages/wsr-types/src/drivers/baseDrivers.d.ts]: OK, so we will need to ask the client to specifically include it

**Pull request merged**